zulip/zulip/__init__.py

```diff
@@ -486,6 +486,11 @@ def __init__(
 
         self.has_connected = False
 
+        server_settings = self.get_server_settings()
+        self.zulip_version: Optional[str] = server_settings.get("zulip_version")
+        self.feature_level: int = server_settings.get("zulip_feature_level", 0)
```

Comment:
Do we want to call this `zulip_feature_level`, or just `feature_level`?  The `zulip_` prefix might be more appropriate for analogy with `zulip_version`.

Reply:
I think `feature_level` will be more concise and it corresponds to what we have in `version.py`